WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteAssemblyWizardContent.swift

```diff
@@ -42,10 +45,15 @@ final class SiteAssemblyWizardContent: UIViewController {
     /// - Parameters:
     ///   - creator: the in-flight creation instance
     ///   - service: the service to use for initiating site creation
+    ///   - quickStartSettings: the UserDefaults helper for quick start settings
     ///   - onDismiss: the closure to be executed upon dismissal
-    init(creator: SiteCreator, service: SiteAssemblyService, onDismiss: ((Blog) -> Void)? = nil) {
+    init(creator: SiteCreator,
+         service: SiteAssemblyService,
+         quickStartSettings: QuickStartSettings? = nil,
+         onDismiss: ((Blog) -> Void)? = nil) {
         self.siteCreator = creator
         self.service = service
+        self.quickStartSettings = quickStartSettings ?? QuickStartSettings()
```

Comment:
The init parameter could be non-optional with a default value of `QuickStartSettings()`, just to prevent the need for this :)

Reply:
Fixed! 72a664b